.jazzy.yml

```diff
@@ -0,0 +1,50 @@
+# jazzy --swift-version 2.2 -m Deferred -g "https://github.com/bignerdranch/Deferred" -a "Big Nerd Ranch" -u "http://bignerdranch.github.io" --module-version=2.0.0 -r "http://bignerdranch.github.io/Deferred/"
```

Comment:
On second thought, looks like this comment line was for copy-paste execution at the CLI. But it's out-of-date now we need so many options passed to jazzy. Nuke the line, or explain how to invoke it with this config file?